src/components/Autocomplete/components/ComboBox/tests/ComboBox.test.tsx

```diff
@@ -50,16 +52,16 @@ describe('<ComboBox/>', () => {
     },
   ];
 
+  const mockProps = {
+    options,
+    selected: [],
+    textField: renderTextField(),
+    onSelect: noop,
+  };
+
   describe('options', () => {
     it('passes options to OptionList', () => {
-      const comboBox = mountWithAppProvider(
-        <ComboBox
-          options={options}
-          selected={[]}
-          textField={renderTextField()}
-          onSelect={noop}
-        />,
-      );
+      const comboBox = mountWithAppProvider(<ComboBox {...mockProps} />);
 
       comboBox.simulate('click');
```

Comment:
I know this is not a change you made in this PR but we probably wanna stick with manually triggering the callback instead of simulating an actual click event on the component:`trigger(comboBox.find('div').first(), 'onClick')`.

Reply:
Simulate won't work in all cases. In this specific case though its mostly that I would argue that using `simulate` is pretty vague as it triggers a click event but its not clear on what.

Reply:
Finding the first div feels fragile to me and others have discouraged it in other PRs. But it makes sense to directly trigger `onClick`.